### ISSUE TITLE
Include demographics in partners applicant csv export 

### ISSUE THREAD

**Issue by sarahlazarich** [metrotranscom/doorway#493]
As a Partner on Doorway, 
I want demographic information from the application to be present in my applicant 
So that I can use that info for TCAC/HUD reporting. 

Notes: 
- Currently, only JAs + Admins get an export with demographics. We want partners to receive the export w/ demo info. 

AC: 
- Applicant csv export for Partners includes answers to demographics questions.

**Comment by sarahlazarich** [comment]
- Can we make this configurable in Core? Yes, time depending 
- Put the feature flag on the jurisdictions table instead of Env variable 
- BUT, Partners can belong to multiple jurisdictions (can use the listing jurisdiction, though)

**Comment by sarahlazarich** [comment]
Just confirming behavior here:
- This should be configurable in Core, but toggled off. Partners csv exports in Core should **not** contain demographics. 
- Once this is complete in Core @ludtkemorgan, can pull into doorway prisma and toggle it on.

**Comment by KrissDrawing** [comment]
@ludtkemorgan it is merged into core. There is a flag `enablePartnerDemographics` in jurisdiction.

QA notes: once merged to doorway with flag set as true It will show demographics when logged as partner (at the end in csv application export). For core it will still return demographics just for an admin.

**Comment by sarahlazarich** [comment]
@KrissDrawing, checking the export in staging, and I'm not seeing the demographics - assuming that's because the flag isn't on in staging?

I also checked the export, and I'm only seeing the race/ethnicity item. Shouldn't we also we the rest of the demographics  questions?
- Which language is most commonly spoken in your home? Please select one:
- Which best describes your gender identity? Please select one:
- Which best describes your sexual orientation or sexual identity? Please select one:

**Comment by sarahlazarich** [comment]
addressing in a follow-up, closing
